### DISCUSSION TITLE
Chain id format

### DISCUSSION THREAD

**Discussion by pociej** [wevm/wagmi#3095] [Ideas]
currently it is number, I would prefer to use hex so no 1 but `0x1` to keep consistency with rest of the code i use. 
Of course its not a big deal to wrap `useNetwork` and parse chain Id but would be nice to be able to control format from config.

**Comment by tmm** [comment]
Thanks for the feedback. This is unlikely to change as we find it to be more friendly as a number.

**Comment by pociej** [comment]
@tmm Yeah but why not give user ability to control the format?
